lib/active_encode/core.rb

```diff
@@ -73,26 +83,41 @@ def remove_output!(output_id)
 
     def reload
       run_callbacks :reload do
-        merge!(self.class.engine_adapter.find(id, cast: self.class))
+        merge!(self.class.engine_adapter.find(id))
+      end
+    end
+
+    def created?
+      !id.nil?
+    end
+
+    # @deprecated
+    def tech_metadata
+      metadata = {}
+      [:width, :height, :frame_rate, :duration, :file_size,
+       :audio_codec, :video_codec, :audio_bitrate, :video_bitrate, :checksum].each do |key|
+        metadata[key] = input.send(key)
       end
     end
 
```

Comment:
i probably missed this discussion: are we making technical metadata optional now?

Reply:
Technical metadata is moving into the input and output classes.  So instead of having a top-level technical metadata hash the input object now has specified attributes.